packages/sales-channel/src/repositories/sales-channel.ts

```diff
@@ -0,0 +1,7 @@
+import { DALUtils } from "@medusajs/utils"
```

Comment:
todo: you can remove this file and create a type in the module types file. you can refer to my last merged pr